src/child/containers/sidebars/star/star.less

```diff
@@ -0,0 +1,54 @@
+.star {
```

Comment:
There is already a `star` class in `favourites.less` for this, can this style (in `star.less`) be removed (and the class in `favourites.less` updated, if required)?